src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -42,33 +54,14 @@ object InputProcessor {
         return None
       }
       val doubleVals = tokens map(_.get)
-      if (doubleVals.length != 6) {
-        println(s"Error, Line $index: Each face of input rock volume is defined by values: a, b, c, d, phi and cohesion")
+      if (doubleVals.length != 7) {
+        println(s"Error, Line $index: Each face of input rock volume is defined by values: strike, dip, x, y, z,"+
```

Comment:
You mean the order of the inputs? Figured geometric inputs first, followed by surficial parameters.